shell/artifacts/Words/source/GamePane.js

```diff
@@ -463,6 +459,24 @@ recipe
         });
       }
     }
+    _onStartGame(e, state) {
```

Comment:
Done. I noted we still do `this._views.get('foo')` for things like setting an instance in a handle. For future, maybe DomParticle should expose a getter for `handles` so that SimpleParticle subclass code can work with the handle without having to start with `this._views`.